src/components/language-switch/language-switch.tsx

```diff
@@ -1,44 +1,91 @@
 import { useI18next } from 'gatsby-plugin-react-i18next';
 import React from 'react';
 import styled from 'styled-components';
-import { Link } from '../links/links';
+import { Chevron } from '../icons/chevron';
 
-const StyledLanguageLink = styled(Link)<{ selected?: boolean }>`
-  margin: 1px 0 1px 12px;
+interface LanguageSwitchProps {
+  translation: any;
+  className?: string;
+  $lightTheme?: boolean;
+  fromNavigation?: boolean;
+}
+
+const StyledNav = styled.nav<{
+  $lightTheme?: boolean;
+  fromNavigation?: boolean;
+}>`
+  transition: color 0.2s;
+  color: ${(props) =>
+    props.fromNavigation || props.$lightTheme
+      ? props.theme.palette.text.default
+      : props.theme.palette.text.header};
+  &:hover {
+    color: ${(props) =>
+      props.fromNavigation
+        ? props.theme.palette.text.header
+        : !props.$lightTheme &&
+          !props.fromNavigation &&
+          props.theme.palette.text.headerHover};
+  }
+`;
+
+const StyledSelection = styled.select<{
+  $lightTheme?: boolean;
+  fromNavigation?: boolean;
+}>`
+  background: none;
+  border: none;
+  text-transform: uppercase;
   font-weight: bold;
   font-size: 14px;
-  line-height: 110%;
+  color: inherit;
+
+  appearance: none;
+  padding-left: 10px;
   cursor: pointer;
-  text-transform: uppercase;
+`;
 
-  ${({ selected }) => (selected ? `color: #668CFF;` : `color: #FFFFFF;`)}
+export const StyledChevron = styled(Chevron)<{
+  $lightTheme?: boolean;
+  fromNavigation?: boolean;
+}>`
+  margin-bottom: 2px;
+  /*margin-right: -6px makes the Chevron clickable*/
```

Comment:
that's because you pull it into the bounding box of the select. I now understand but the comment doesn't help. I also see that the magic number of 6px equals the width of the chevron. It works, and it's okay for now. 

I would have tried to use a `<label>` which can include the icon, some (visual hidden) text for accessibility reasons. Then the label can refer to the select through `label for` or such? Does this work here? It's a very small detail and I only want to give general feedback about those small things. No need to fix right now.

![Screen Shot 2021-11-08 at 10 35 23](https://user-images.githubusercontent.com/1701755/140718179-6d286fea-2731-4924-80dd-25710c0aaf57.png)